tsl/src/remote/connection.c

```diff
@@ -288,7 +288,16 @@ fill_result_error(TSConnectionError *err, int errcode, const char *errmsg, const
 	const char *sqlstate;
 
 	if (NULL == err || NULL == res || NULL == entry)
+	{
+		if (err)
+		{
+			MemSet(err, 0, sizeof(*err));
+			err->errcode = errcode;
+			err->msg = errmsg;
+			err->nodename = "";
+		}
```

Comment:
Why isn't this part covered by the tests, can we add something for it as well?

Reply:
This is done mostly as precaution, because otherwise error object will be left uninitialized in some calls